newIDE/app/src/BrowserApp.js

```diff
@@ -99,4 +99,4 @@ export const create = (authentification: Authentification) => {
   );
 
   return app;
-};
+};
```

Comment:
nitpicking: Configure your editor to add an line at the end of the files

Reply:
Oh! My mistake. While comparing files to remove unrelated changes, I was comparing the files from the master using online GitHub editor and my local files. In the online editor, it does not show the end line, so I thought end line is not there in the master, so I removed it. I'll fix it.